olive/passes/onnx/conversion.py

```diff
@@ -113,6 +113,37 @@ def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassCon
 
     def _run_for_config(
         self, model: PyTorchModelHandler, data_root: str, config: Dict[str, Any], output_model_path: str
+    ) -> Union[CompositeModelHandler, DistributedOnnxModelHandler, ONNXModelHandler]:
+        output_model = self._run_for_config_internal(model, data_root, config, output_model_path)
+
+        if model.hf_config:
+            if isinstance(output_model, CompositeModelHandler):
```

Comment:
seems these isinstance check already exist in Pass.run. Could we just call save_hf_* simply here?

Reply:
Pass has branches based on input model. Here's the branching is based on output model.